src/main/java/nl/novi/techiteasycontroller/models/WallBracket.java

```diff
@@ -0,0 +1,69 @@
+package nl.novi.techiteasycontroller.models;
+
+import jakarta.persistence.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "wall_bracket")
+public class WallBracket {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String size;
+    private Boolean adjustable;
+    private String name;
+    private Double price;
+
+    @ManyToMany (mappedBy = "wallBrackets")
+    private List<Television> televisions = new ArrayList<Television>();
```

Comment:
Hier ook "= new ArrayList<Televisions>()" weglaten

Reply:
Goed opgemerkt, Bo. Aan de andere kant van de relatie, de List<WallBracket>  in de Television class, moet wel "= new ArrayList()" hebben, omdat je daar op een gegeven moment een WallBracket wilt toevoegen aan de lijst met de ".add()" methode. Als je dan niet een ArrayList, maar null hebt, dan heb je een probleem.